tests/Platforms/SQLServerPlatformTest.php

```diff
@@ -46,4 +46,26 @@ public function testGeneratesTypeDeclarationForDateTimeTz(): void
     {
         self::assertEquals('DATETIMEOFFSET(6)', $this->platform->getDateTimeTzTypeDeclarationSQL([]));
     }
+
+    /**
+     * @param array<string, mixed> $column
+     *
+     * @dataProvider sqlDeclarationDataProvider
+     */
+    public function testAsciiSQLDeclaration(string $expectedSql, array $column): void
```

Comment:
This could be moved to the `AbstractPlatformTest` and cover all platforms. Only the data provider will have to be overridden here (see other platform tests).